__tests__/app.test.js

```diff
@@ -121,3 +122,33 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET:200 Responds with an array of comments for the given article_id of which each comment should have the following properties", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(2);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        body.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("GET:404 Responds with a 404 error message when article_id does not match any existing article_id in the database ", () => {
+    return request(app)
+      .get("/api/articles/2325/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid article ID");
+      });
+  });
```

Comment:
A couple tests missing here:

- What if the article id is not a number?
- What if the article does exist but has no comments? eg. article id 2. In this case, we would want a 200 status and the empty array returned